apps/web-giddh/src/app/vouchers/template-edit-dialog/template-edit-dialog.component.ts

```
@@ -1,39 +1,225 @@
+  /**
+ * Replaces new line characters with <br /> tags in specific template fields.
+ *
+ * @param {*} template The template object to update
+ * @returns {*} The updated template object
+ * @memberof TemplateEditFilterComponent
+ */
+  public newLineToBR(template): void {
+    template.sections['footer'].data['message1'].label = template.sections['footer'].data['message1'].label ? template.sections['footer'].data['message1'].label?.replace(/(?:\r\n|\r|\n)/g, '<br />') : template.sections['footer'].data['message1'].label = '';
+    template.sections['footer'].data['companyAddress'].label = template.sections['footer'].data['companyAddress'].label ? template.sections['footer'].data['companyAddress'].label?.replace(/(?:\r\n|\r|\n)/g, '<br />') : template.sections['footer'].data['companyAddress'].label = '';
+    template.sections['footer'].data['slogan'].label = template.sections['footer'].data['slogan'].label?.replace(/(?:\r\n|\r|\n)/g, '<br />');
+    return template;
```

Biome (2.1.2)

[error] The function should not return a value because its return type is void.

The function is here:

'void' signals the absence of value. The returned value is likely to be ignored by the caller.

(lint/correctness/noVoidTypeReturn)